src/nft/components/details/detailsV2/DataPage.tsx

```diff
@@ -47,7 +47,7 @@ export const DataPage = ({ asset }: { asset: GenieAsset }) => {
       <DataPageHeader />
       <ContentContainer>
         <LeftColumn>
-          <DataPageTraits asset={asset} />
+          {!!asset.traits?.length && <DataPageTraits asset={asset} />}
```

Comment:
traits is possibly undefined so `!!` covers both that it is defined and has a value greater than 0

Reply:
@grabbou see this comment for using !! over > 0

Reply:
That makes sense to me, however, still, it's a question what we consider more readable. Personally, I would still do `> 0` to embrace optional chaining (at the disadvantage of having `length` being undefined, converted to `NaN` and compared against 0, which always returns false). Your approach is also valid, in a way that it will convert both `undefined` and `0` (when `traits` is defined). I let you pick the right choice.